#!/usr/bin/env python
PACKAGE = "Robotics_project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

int_enum = gen.enum([ gen.const("Euler",      int_t, 0, "Euler integration"),
                      gen.const("RK",       int_t, 1, "Runge-Kutta integration")],
                      "Enum of formats")
gen.add("int_choice", int_t, 1, "Format of count", 0, 0, 1, edit_method=int_enum)


exit(gen.generate(PACKAGE, "odom", "integration_methods"))